src/components/repo-list/RepoList.js

```diff
@@ -0,0 +1,67 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import moment from 'moment';
+import { Link as RouterLink } from 'react-router-dom';
+import { Box, Divider, Flex, Heading, Link, Text } from '@chakra-ui/react';
+import { StarIcon } from '@chakra-ui/icons';
+
+const Repo = ({ repo, user }) => {
+  const {
+    description,
+    language,
+    name,
+    pushed_at: pushedAt,
+    stargazers_count: stars,
+  } = repo;
+  const updatedAtForDisplay = moment(pushedAt).fromNow();
+  const starsForDisplay = stars ? (
+    <Flex align="baseline">
+      <StarIcon boxSize="14px" />
+      <Text fontSize="18px" ml="4px">
+        {stars}
+      </Text>
+    </Flex>
+  ) : null;
+
+  return (
+    <Box mb="12px">
+      <Flex align="baseline" justify="space-between">
+        <Link as={RouterLink} to={`${user}/repo/${name}`}>
+          <Text fontSize="24px">{name}</Text>
+        </Link>
+        {starsForDisplay}
+      </Flex>
+
+      <Text fontSize="14px" mb="10px">
+        {description}
+      </Text>
+
+      <Text fontSize="14px">{language}</Text>
```

Comment:
I think we can expand on this language thing. maybe `javascript` would be black text and rounded yellow background color and so on.

Reply:
Sounds good, let's get this PR in first to avoid merge conflicts. I will make a ticket for this.